src/node.cc

```diff
@@ -3861,6 +4008,7 @@ static void CheckIfAllowedInEnv(const char* exe, bool is_env,
     "--force-fips",
     "--openssl-config",
     "--icu-data-dir",
+    "--abort_on_unhandled_rejection",
```

Comment:
Please also add this to the documented list of options that are allowed in `NODE_OPTIONS` in `doc/api/cli.md`.

Reply:
@richardlau done!